test/cql-pytest/cassandra_tests/validation/operations/multi_column_relation_tuples_null_check.py

```diff
@@ -0,0 +1,16 @@
+from cassandra_tests.porting import *
+
+def test(cql, test_keyspace):
+    with create_table(cql, test_keyspace, "(a int, b int, c int, d int, primary key (a, b, c, d))") as table:
```

Comment:
Please try to make this test a minimal test that will fail before you patch and pass afterwards. Do we really need 6 different bound variables in this test? 

Also please:
1. Give the test a name, not "test" ;-)
2. Add a comment in front of the test explaining what it checks, and most importantly mentioning that it reproduces #13217 (this information will be useful in case of regression or a missing backport).